nservicebus/upgrades/5to6.md

```diff
@@ -45,15 +45,15 @@ WARNING: Do not `return null` from the message handlers. A `null` will result in
 
 snippet:5to6-messagehandler
 
+For a step by step upgrade of existing handlers or sagas, see: [Migrate existing handlers/sagas](/nservicebus/upgrades/5to6-migrate-existing-handlers.md).
```

Comment:
So in a document about **migrating** from V5 to V6, in a section named **Message handlers**, we need a subheading called **Migrating existing handlers**?

If that adds understanding, that's cool. I question the need to expand the (already mammoth) TOC any further though.